configureWorkspace/configure.ts

```diff
@@ -153,15 +153,17 @@ LABEL Name=${serviceName} Version=${version}
 EXPOSE ${port}
 
 WORKDIR /app
-ADD . /app
+COPY . .
+
+# install dependencies if dependencies file is present
+RUN python3 -m pip install -r requirements.txt || echo ‘No requirements.txt - skipping pip install’
```

Comment:
I wouldn't use opening and closing quotes and just stick with single quotes (`'`).